apertium-kaa.kaa.lexc

```diff
@@ -714,6 +714,16 @@ LEXICON NP-COG-MF
 %<np%>%<cog%>%<mf%>: NP-COMMON ;
 %<np%>%<cog%>%<mf%>%<pl%>:%>%{L%}%{A%}r NP-COMMON ;
 
+
+LEXICON NP-PAT-VICH
+
+! For patronyms ending with -vich
+! Their feminine equivalets end with -ovna /I.S./
+
+%<np%>%<pat%>%<m%>:vich%{%} NP-COMMON ;
```

Comment:
What is `%{%}` for?  You can probably remove it.

Reply:
removed